Content.Shared.Database/LogType.cs

```diff
@@ -3,99 +3,413 @@ namespace Content.Shared.Database;
 // DO NOT CHANGE THE NUMERIC VALUES OF THESE
 public enum LogType
 {
-    Unknown = 0, // do not use
+    /// <summary>
+    /// Test logs. DO NOT USE!!!
+    /// </summary>
+    Unknown = 0,
     // DamageChange = 1
+
+    /// <summary>
+    /// A player dealt damage to an entity.
+    /// </summary>
     Damaged = 2,
+
+    /// <summary>
+    /// A player healed an entity.
+    /// </summary>
     Healed = 3,
+
+    /// <summary>
+    /// A player slipped on an entity.
+    /// </summary>
     Slip = 4,
+
+    /// <summary>
+    /// Station event was added or announced.
+    /// </summary>
     EventAnnounced = 5,
+
+    /// <summary>
+    /// Game rule was added or started.
+    /// </summary>
     EventStarted = 6,
-    EventRan = 16,
+    // EventRan = 16,
+
+    /// <summary>
+    /// Game rule was stopped.
+    /// </summary>
     EventStopped = 7,
+
+    /// <summary>
+    /// A player used a verb on an entity.
+    /// </summary>
     Verb = 19,
+
+    /// <summary>
+    /// An evacuation shuttle was called.
+    /// </summary>
     ShuttleCalled = 8,
+
+    /// <summary>
+    /// An evacuation shuttle was recalled.
+    /// </summary>
     ShuttleRecalled = 9,
+
+    /// <summary>
+    /// Explosive depressurization related interactions.
+    /// </summary>
     ExplosiveDepressurization = 10,
+
+    /// <summary>
+    /// A player or entity was respawned.
+    /// </summary>
     Respawn = 13,
+
+    /// <summary>
+    /// A player joined station on round start.
+    /// </summary>
     RoundStartJoin = 14,
+
+    /// <summary>
+    /// A player joined station after round start.
+    /// </summary>
     LateJoin = 15,
+
+    /// <summary>
+    /// Chemical reactions related interactions.
+    /// </summary>
     ChemicalReaction = 17,
+
+    /// <summary>
+    /// Reagent effects related interactions.
+    /// </summary>
     ReagentEffect = 18,
+
+    /// <summary>
+    /// Canister valve was opened or closed.
+    /// </summary>
     CanisterValve = 20,
+
+    /// <summary>
+    /// Release pressure on the canister was changed.
+    /// </summary>
     CanisterPressure = 21,
+
+    /// <summary>
+    /// Canister purged its contents into the environment.
+    /// </summary>
     CanisterPurged = 22,
+
+    /// <summary>
+    /// Tank was ejected from the canister.
+    /// </summary>
     CanisterTankEjected = 23,
+
+    /// <summary>
+    /// Tank was inserted into the canister.
+    /// </summary>
     CanisterTankInserted = 24,
+
+    /// <summary>
+    /// A player tried to disarm an entity.
+    /// </summary>
     DisarmedAction = 25,
+
+    /// <summary>
+    /// A player knocked down an entity on the floor.
+    /// </summary>
     DisarmedKnockdown = 26,
-    AttackArmedClick = 27,
-    AttackArmedWide = 28,
-    AttackUnarmedClick = 29,
-    AttackUnarmedWide = 30,
+    // AttackArmedClick = 27,
+    // AttackArmedWide = 28,
+    // AttackUnarmedClick = 29,
+    // AttackUnarmedWide = 30,
```

Comment:
Not sure we should comment out unused log types like this. While it's unlikely, it's possible for downstreams to still refer to these and that'd just be annoying for them.